guides/source/active_record_validations.md

```diff
@@ -26,9 +28,9 @@ end
 ```
 
 ```irb
-irb> Person.create(name: "John Doe").valid?
+irb> Person.new(name: "John Doe").valid?
 => true
-irb> Person.create(name: nil).valid?
+irb> Person.new(name: nil).valid?
 => false
 ```
 
```

Comment:
For the paragraph below:
1. The use of "our" and "us" reads odd. It can as simple as  "The `Person` is not valid without a `name` attribute.?
2. The 2nd sentence is not true now. With the use of `new` neither are persisted to the database without `save` right?

Reply:
Good catch, thank you!